test/unit/spec/EnrollU2F_spec.js

```diff
@@ -103,7 +123,46 @@ function (Okta,
     });
 
     Expect.describe('Enroll factor', function () {
+      itp('shows error if wrong browser', function () {
+        delete window.u2f;
+        mockFirefox(false);
+
+        return setup().then(function (test) {
+          expect(test.form.errorHtml()).toHaveLength(1);
+          expect(test.form.errorHtml().html()).toEqual(Okta.loc('u2f.error.notSupportedBrowser', 'login'));
```

Comment:
We should check against the actual text instead of calling ```Okta.loc```. Reason is because if there is an error in ```Okta.loc``` or if someone messes up those keys, we will not know about it
Same applies to all the other places you are using ```Okta.loc``` in this test